Source/Private/ASElementMap.m

```diff
@@ -54,8 +54,8 @@ - (instancetype)initWithSections:(NSArray<ASSection *> *)sections items:(ASColle
       }
       s++;
     }
-    for (NSDictionary *supsOfKind in [_supplementaryElements objectEnumerator]) {
-      [supsOfKind enumerateKeysAndObjectsUsingBlock:^(NSIndexPath *_Nonnull indexPath, ASCollectionElement * _Nonnull element, BOOL * _Nonnull stop) {
+    for (NSDictionary *supplementariesForKind in [_supplementaryElements objectEnumerator]) {
+      [supplementariesForKind enumerateKeysAndObjectsUsingBlock:^(NSIndexPath *_Nonnull indexPath, ASCollectionElement * _Nonnull element, BOOL * _Nonnull stop) {
```

Comment:
Per a nit mentioned by @garrettmoon in [#3073](https://github.com/facebook/AsyncDisplayKit/pull/3073#discussion_r103034371).